tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestPersonNameFormatter.java

```diff
@@ -94,7 +94,7 @@ public void TestBasic() {
         check(personNameFormatter, sampleNameObject1, "length=short; style=formal; usage=addressing", "John B. Smith");
         check(personNameFormatter, sampleNameObject2, "length=short; style=formal; usage=addressing", "John Smith");
         check(personNameFormatter, sampleNameObject1, "length=long; style=formal; usage=addressing", "Mr. John Bob Smith Barnes Pascal Jr.");
-        check(personNameFormatter, sampleNameObject3, "length=monogram; style=formal; usage=addressing", "JS");
+        check(personNameFormatter, sampleNameObject3, "length=long; style=formal; usage=monogram", "J* B*S*"); // TODO: This seems wrong....
```

Comment:
This result looks odd. But I thought it best to get this PR merged first (to unblock other work) before investigating/debugging this, if it is indeed an error.

Reply:
Agreed. I can look at that after it is in.